ts/features/idpay/details/screens/InitiativeDetailsScreen.tsx

```diff
@@ -184,6 +185,9 @@ const InitiativeDetailsScreen = () => {
                     size={5}
                   />
                   <VSpacer size={32} />
+                  <InitiativeDiscountSettingsComponent
+                    initiative={initiative}
+                  />
```

Comment:
Just a little bit of spacing

```suggestion
                  <InitiativeDiscountSettingsComponent
                    initiative={initiative}
                  />
                  <VSpace size={16} />
```